pkg/config/settings.go

```diff
@@ -27,8 +27,9 @@ import (
 )
 
 const (
-	EnvNameCrYaml string = "CR_YAML"
-	NodeAgentTag         = "1.0.0"
+	EnvNameIBMBlockCSICrYaml    = "IBMBlockCSI_CR_YAML"
+	EnvNameHostDefinitionCrYaml = "HostDefinition_CR_YAML"
+	NodeAgentTag                = "1.0.0"
```

Comment:
I know it's existing code but why do we have this field?

Reply:
legacy dead code, should be removed

Reply:
I will remove the NodeAgentTag variable

Reply:
done